src/main/java/duke/Parser.java

```diff
@@ -1,4 +1,4 @@
-
+package duke;
```

Comment:
I understand that you might only have one package, but can consider importing a specific class in the package like, duke.Tasklist and duke.Storage